doc/source/user_guide/hpc/launch_mapdl_entrypoint.rst

```diff
@@ -0,0 +1,235 @@
+
+.. _ref_pymapdl_interactive_in_cluster_hpc:
+
+.. _ref_pymapdl_interactive_in_cluster_hpc_from_login:
+
+
+Interactive MAPDL instance launched from the login node
+=======================================================
+
+Starting the instance
+---------------------
+
+If you are already logged in an login node, it is also
+possible to launch an MAPDL instance as an SLURM job and
+connect to it.
+This can be accomplished running the following commands
+in your login node.
+
+.. code:: pycon
+
+    >>> from ansys.mapdl.core import launch_mapdl
+    >>> mapdl = launch_mapdl(launch_on_hpc=True)
+
+PyMAPDL submits a job to the scheduler using the appropriate commands.
+In case of SLURM, it uses ``sbatch`` command with the ``--wrap`` argument
+to pass the MAPDL command line need to start.
+Other scheduler arguments can be specified using ``scheduler_options``
+argument as a Python :class:`dict`:
+
+.. code:: pycon
+
+    >>> from ansys.mapdl.core import launch_mapdl
+    >>> scheduler_options = {"nodes": 10, "ntasks-per-node": 2}
+    >>> mapdl = launch_mapdl(launch_on_hpc=True, nproc=20, scheduler_options=scheduler_options)
+
+
+.. note::
+    PyMAPDL cannot infer the number of CPU you are requesting from the scheduler,
+    hence you need to specify this value using the ``nproc`` argument.
+
+The double minus (``--``) common in the long version of some scheduler commands
+are added automatically if PyMAPDL detects it is missing and the specified
+command is long (length more than 1 character).
+For instance ``ntasks-per-node`` argument is submitted as ``--ntasks-per-node``.
+
+or as a single Python string (:class:`str`):
+
+.. code:: pycon
+
+    >>> from ansys.mapdl.core import launch_mapdl
+    >>> scheduler_options = "-N 10"
+    >>> mapdl = launch_mapdl(launch_on_hpc=True, scheduler_options=scheduler_options)
+
+.. warning::
+    Because PyMAPDL is already using the ``--wrap`` argument, this argument
+    cannot be used again.
+
+The values of each scheduler argument are wrapped with single quotes (`'`).
+This might cause parsing issues which can cause the job to fail after successful
+submission.
+
+It should be noticed also that PyMAPDL passes all the environment variables of the
+user to the new job and to the MAPDL instance.
+This is normally convenient, since there are many environmental variables which are
+needed to run the job or MAPDL.
```

Comment:
```suggestion
PyMAPDL passes all the environment variables of the
user to the new job and to the MAPDL instance.
This is usually convenient because many environmental variables are
needed to run the job or MAPDL command.
```